multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -435,7 +490,11 @@ def infer_column_types(
         elif is_numerical_column(data[col_name], valid_data[col_name]):  # Infer numerical column
             column_types[col_name] = NUMERICAL
         elif is_image_column(data[col_name], col_name=col_name, image_type=IMAGE_PATH):  # Infer image-path column
-            column_types[col_name] = IMAGE_PATH
+            # Check if it is document image or not.
+            if is_document_image_column(data[col_name], col_name=col_name):
```

Comment:
What's the approximate type check time increase here? Doing OCR on 30 images for every image task seems heavy.

Reply:
do you have a suggested number?

Reply:
set to 10 as discussed.